src/app/core/parsers/zod-action-event-trigger-parser.service.spec.ts

```diff
@@ -56,3 +63,9 @@ describe(ZodActionTriggerEventParser.name, () => {
     })
   })
 })
+
+function createMockOfActionTriggerParser(): ActionTriggerParser {
```

Comment:
Done

Reply:
You say "done", yet no changes has happened. Are they yet to be pushed?

Reply:
Ahh, you renamed the local variables instead of the method :) 
`createActionTriggerParserMock` is the method name. The result is still just an actionTriggerParser. 
So the usage will look like this:
```
const actionTriggerParser: ActionTriggerParser = createActionTriggerParserMock()
```

`create` is a verb and verbs should not be used in variables name. Instead they should be used in methods names. This is because verbs indicates an action; that something will happen. We don't expect anything to happen when we access a variable. We just want to get whatever state that variable has. When calling a method however, we expect the method to do something, that something will happen. Hence we don't use verbs for naming variables, but we do use verbs for naming methods.

Reply:
Just a note: `createMockOf<Class>` is the naming convention used for most of the mocking functions. I think we should only use one naming convention.

Reply:
Done